src/main/java/net/mcreator/generator/LocalizationUtils.java

```diff
@@ -83,6 +80,14 @@ private static void generateJSONLanguageFiles(Generator generator, Workspace wor
 
 	public static void extractLocalizationKeys(Generator generator, GeneratableElement element,
 			@Nullable List<?> localizationkeys) {
+		processDefinitionToLocalizationKeys(generator, element, localizationkeys)
+				.forEach((k, v) -> addLocalizationEntry(generator, localizationkeys, v, k));
+	}
+
+	static Map<String, Object> processDefinitionToLocalizationKeys(Generator generator,
+			GeneratableElement element, @Nullable List<?> localizationkeys) {
+		HashMap<String, Object> keysToEntries = new HashMap<>();

```

Comment:
This map is only created but never assigned to. `processDefinitionToLocalizationKeys` also still also extracts keys (addLocalizationEntry), not just processes them.

I would also rename `extractLocalizationKeys` to `generateLocalizationKeys` or `addLocalizationKeysToWorkspace`

Reply:
Seems I forgot to include these couple of lines when making the first commit and did not notice later :man_facepalming:

Reply:
:heavy_check_mark: Fixed